server/api.py

```diff
@@ -27,3 +35,15 @@ def api_login(request, credentials: Credentials):
         return 200, {"message": "Login successful"}
     else:
         return 403, {"message": "Invalid credentials"}
+
+
+@api.post("/registration", response={200: Response, 400: Response})
+def register_player(request, player: PlayerSchema):
+    user = request.user
+    if Player.objects.filter(user=user).exists():
```

Comment:
Since you are just returning a 400, you could just do it in a `try ... except ...` block, may be? :thinking: